### PR TITLE
feature/add_two_scripts_to_start_mailserver

### PR DESCRIPTION
**Pull request by McBlankenburg**

### CODE REVIEW

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    echo "Error: application.properties file not found."
+    exit 1
+fi
+
+# Checking if docker daemon is running
+if ! docker info > /dev/null 2>&1; then
+    echo "Error: Failed to connect to Docker daemon. Please check if Docker daemon is running and accessible."
+    echo "To solve this problem, try using the command depending on your system:"
+    echo "    -> sudo systemctl start docker"
+    echo "    -> sudo service docker start"
```
Comment (dawciobiel) [inline scripts/mailserver-launch-starter/mailserver-launch-starter.bash:28]:
```suggestion
    echo "    sudo systemctl start docker"
    echo "    sudo service docker start"
```

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    echo "and start script again."
+    exit 1
+fi
+
+# Check if the container already exists, if so, delete it
```
Comment (dawciobiel) [inline scripts/mailserver-launch-starter/mailserver-launch-starter.bash:33]:
```suggestion
# Delete the container if it already exists.
```

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+if docker ps -a --format "{{.Names}}" | grep -q "^$CONTAINER_NAME$"; then
+    echo "Deleting old container: $CONTAINER_NAME"
```
Comment (dawciobiel) [inline scripts/mailserver-launch-starter/mailserver-launch-starter.bash:35]:
```suggestion
    echo "Deleting container: $CONTAINER_NAME"
```

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+# Starting a new container
+output=$(docker run -d \
+--name "$CONTAINER_NAME" \
+-p "$MAILDEV_WEB_PORT":1080 \
+-p "$MAILDEV_OUTGOING_PORT":1025 \
+-e MAILDEV_WEB_USER="$USERNAME" \
+-e MAILDEV_WEB_PASS="$PASSWORD" \
+maildev/maildev 2>&1)
+
+# Checking error codes and displaying the appropriate message based on it
```
Comment (dawciobiel) [inline scripts/mailserver-launch-starter/mailserver-launch-starter.bash:49]:
```suggestion
# Checking error codes and displaying the appropriate message
```

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    elif [[ $output == *"port is already allocated"* ]]; then
+        echo "Error: Port conflict. The specified port is already allocated by another container."
+    else
+        echo "Error: Failed to create a new container: $CONTAINER_NAME, Unknown reason."
+    fi
+    exit 1
+fi
+
+echo "Created new container: $CONTAINER_NAME."
+echo "___________________________________"
```
Comment (dawciobiel) [inline scripts/mailserver-launch-starter/mailserver-launch-starter.bash:69]:
```suggestion

```

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

File: scripts/mailserver-launch-starter/mailserver-launch-starter.bash (hunk 3 above)
Comment (McBlankenburg) [reply scripts/mailserver-launch-starter/mailserver-launch-starter.bash:35]: I can replace the word "old" with "previous" but I think it should be one of these two to know that it is about a previously created container not necessarily by this script

**Pull request closed without merging**